docs/courses/README.md

```diff
@@ -1,28 +1,104 @@
 # Contributing to LibreLingo courses
 
-LibreLingo courses consist of files that contain course content. These files
-are organized in a specific directory structure.
-[Learn how to edit or create courses](editing-courses.md)!
+This document describes how LibreLingo's course system works and how to contribute to course content.
 
+Understanding this document is the first step if you want to:
+- edit or improve an existing course
+- contribute a new accepted answer
+- create a new course from scratch
+
+This is also a good resource if you just want to learn about the course system in general.
+
+---
+
+**NOTE**: If you are not clear about anything on this document feel free to post on [Discussions](https://github.com/kantord/LibreLingo/discussions) so that the community can help you further.
+
+---
+
+**Table of Contents:**
+- [Getting started](#getting-started)
+- [Basics](#basics)
+  - [Terminology](#terminology)
+  - [Exploring the courses](#exploring-courses)
+  - [Things to do for new contributors](#things-new-contributors)
+- [Courses currently open for contributions](#courses-open)
+  - [For English speakers](#courses-from-english)
+- [Caution](#caution)
+  - [Languages that don't use the Latin alphabet](#non-latin-alphabet)
+
+## Getting started
+
+- Totally new to LibreLingo? Head to the [Basics](#basics) section! It will give you a general understanding about things.
+- Want to edit or improve existing courses? Check out [this page about editing courses](editing-courses.md).
+- Want to create a new course? Read out [this guide on how to create new courses](creating-courses.md). (Advanced)
+
+
+## Basics
+
+### Terminology
+
+You'll need to have an idea on basic terminology used in LibreLingo:
+
+- A **Course** is a set of Modules. There is typically one course for each language pair. e.g. Spanish for English speakers, Japanese for Italian speakers, etc.
+- A **Module** is a set of Skills.
+- A **Skill** is a small unit of course material. In the web app, the user will typically practice a Skill at a time.
+
+These concepts represent a hierarchical structure to LibreLingo course material: `Courses > Modules > Skills`
+
+- The **target language** is the language the course is meant to teach.
+- The **source langauge** is a language that users of the course are assumed to know.
+
+The course directory name is usually structured in `<destination language>-from-<target audience's language>`. e.g. `french-from-english` teaches French to English speakers.
+
+<a id="exploring-courses"></a>
+### Exploring the courses
+
+To have a better understanding of how courses work, you can explore how the course files are organized on this repository.
+
+On the `courses` directory of [this repository](https://github.com/kantord/LibreLingo/tree/main/courses) you will find courses. Under each course you'll find modules. And inside each module you'll find skills, which are saved as individual .yaml files.
```

Comment:
```suggestion
You will find courses in the `courses` directory of [LibreLingo's repository](https://github.com/kantord/LibreLingo/tree/main/courses). In each course you'll find modules. And in each module you'll find skills, which are saved as individual .yaml files.
```

Reply:
OK. So considering the previous changes this becomes:
```suggestion
You will find courses in the `courses` directory of [LibreLingo's repository](https://github.com/kantord/LibreLingo/tree/main/courses). The course directory name is usually structured in `<destination language>-from-<target audience's language>`. e.g. `french-from-english` teaches French to English speakers. In each course you'll find modules. And in each module you'll find skills, which are saved as individual .yaml files.
```